gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/LocalGcsHelper.java

```diff
@@ -1,68 +0,0 @@
-/*
```

Comment:
Why did you remove this file? The concept of a `LocalXXXHelper` is common and makes tesing a code that use serivce XXX easy (by getting an instance of that service from the helper's returned service options).